frontend/src/components/App/TopBar.tsx

```diff
@@ -395,6 +409,12 @@ export function PureTopBar({
       ),
     },
   ];
+
+  const visibleMobileActions = processAppBarActions(
+    allAppBarActionsMobile,
+    appBarActionsProcessors
+  ).filter(action => React.isValidElement(action) || typeof action === 'function');
```

Comment:
I'm always getting empty array here. It's checking if action is valid element but it has to check an action property inside action object so either `React.isValidElement(action.action)` or 

```tsx
filter(({action}) => React.isValidElement(action) || typeof action === 'function');
```

Reply:
made the change, looks like this now:

![image](https://github.com/user-attachments/assets/a56e97d9-4184-44e7-9a4c-a4514f3830ac)

can you confirm if the home context still shows the three dots? I'm trying to see if it's worth keeping the second condition mainly for things like the dynamic cluster plugin, although it's being buggy for me atm:

![image](https://github.com/user-attachments/assets/064b31c5-f77e-445b-a6f8-05845234ddec)